### DISCUSSION TITLE
"How to reset state" guide page has invalid codesandbox links

### DISCUSSION THREAD

**Discussion by linardsblk** [pmndrs/zustand#2889] [General]
I am getting "Sandbox not found" on all links ont his page https://zustand.docs.pmnd.rs/guides/how-to-reset-state

Really wanted to see how to reset with immer since it keeps merging.

**Comment by dai-shi** [comment]
Unfortunately, we can't do anything about it. I hope the code snippets would help.

**Comment by dai-shi** [comment]
#3013 @aabuhijleh

I think we need to find and ask the person who created it...
